src/pyOpenMS/tests/unittests/test_MSSpectrumAndRichSpectrum.py

```
@@ -1,12 +1,74 @@
 import unittest
 import os
-
 import pyopenms
-
+from pyopenms import Precursor, PeptideIdentification, PeptideHit, AASequence
+import numpy as np
 class TestMSSpectrumAndRichSpectrum(unittest.TestCase):
 
     def setUp(self):
         dirname = os.path.dirname(os.path.abspath(__file__))
```

Ruff (0.8.2)

Local variable `dirname` is assigned to but never used

Remove assignment to unused variable `dirname`

(F841)